include/wx/ribbon/buttonbar.h

```diff
@@ -58,68 +58,68 @@ class WXDLLIMPEXP_RIBBON wxRibbonButtonBar : public wxRibbonControl
     virtual wxRibbonButtonBarButtonBase* AddDropdownButton(
                 int button_id,
                 const wxString& label,
-                const wxBitmap& bitmap,
+                const wxBitmapBundle& bitmap,
                 const wxString& help_string = wxEmptyString);
 
     virtual wxRibbonButtonBarButtonBase* AddHybridButton(
                 int button_id,
                 const wxString& label,
-                const wxBitmap& bitmap,
+                const wxBitmapBundle& bitmap,
                 const wxString& help_string = wxEmptyString);
 
     virtual wxRibbonButtonBarButtonBase* AddToggleButton(
                 int button_id,
                 const wxString& label,
-                const wxBitmap& bitmap,
+                const wxBitmapBundle& bitmap,
                 const wxString& help_string = wxEmptyString);
 
     virtual wxRibbonButtonBarButtonBase* AddButton(
                 int button_id,
                 const wxString& label,
-                const wxBitmap& bitmap,
-                const wxBitmap& bitmap_small = wxNullBitmap,
-                const wxBitmap& bitmap_disabled = wxNullBitmap,
-                const wxBitmap& bitmap_small_disabled = wxNullBitmap,
+                const wxBitmapBundle& bitmap,
+                const wxBitmapBundle& bitmap_small = wxNullBitmap,
+                const wxBitmapBundle& bitmap_disabled = wxNullBitmap,
+                const wxBitmapBundle& bitmap_small_disabled = wxNullBitmap,
```

Comment:
Sorry, this doesn't make sense, the bundle should contain both the normal and the small versions of the image. I.e. we need to keep the original function for compatibility, but the new one should take only `bundle` and `bundle_disabled` and not 4 bundles.

Reply:
Done.